regolith/helpers/Google_Calendar_API/google_calendar_api.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import print_function
+import datetime
+import os
+import pathlib
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+
+
+def add_to_calendar(event):
```

Comment:
since the function is called `add_to_calendar` we probably want it to take an optional argument of `calendar_type` with a default of `"google"` and then wrap everything in `if calendar_type = "google"` 

I never did a google auth before though so I may not understand exactly what it is doing.